--- ../../MercuryDPM/MercurySource/Kernel/VTKWriter/BaseVTKWriter.h	2021-02-18 18:01:37.807774887 +1100
+++ BaseVTKWriter.h	2021-02-18 20:41:38.716116377 +1100
@@ -113,9 +113,9 @@
 
     // write output file header
     file << "<?xml version=\"1.0\"?>\n";
-    file << "<!-- time " << handler_.getDPMBase()->getTime() << "-->\n";
-    file << "<VTKFile type=\"UnstructuredGrid\" version=\"0.1\" byte_order=\"LittleEndian\">\n";
-    file << "<UnstructuredGrid>\n";
+    //file << "<!-- time " << handler_.getDPMBase()->getTime() << "-->\n";
+    file << "<VTKFile type=\"PolyData\" version=\"0.1\" byte_order=\"LittleEndian\">\n";
+    file << "<PolyData>\n";
     return file;
 }
 
@@ -132,7 +132,7 @@
     file << "  </DataArray>\n";
     file << "</Cells>\n";
     file << "</Piece>\n";
-    file << "</UnstructuredGrid>\n";
+    file << "</PolyData>\n";
     file << "</VTKFile>\n";
     // close output file
     file.close();
